button/button.css

```diff
@@ -0,0 +1,108 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600;700;900&display=swap");
+
+:root {
```

Comment:
It would be better if the variables are in a separate css file if they are repeated or atleast in the main css file, and import it

Reply:
Will do this when I refactor the whole code.